bci_main.py

```diff
@@ -166,11 +158,11 @@ def _clean_up_session(display, daq, server):
     parser.add_argument('-p', '--parameters', default='bcipy/parameters/parameters.json',
                         help='Parameter location. Must be in parameters directory. Pass as parameters/parameters.json')
     parser.add_argument('-u', '--user', default='test_user')
-    parser.add_argument('-t', '--type', default=1,
+    parser.add_argument('-t', '--task', default='RSVP Calibration',
                         help=f'Task type. Options: ({task_options})')
-    parser.add_argument('-m', '--mode', default='RSVP',
-                        help='BCI mode. Ex. RSVP, MATRIX, SHUFFLE')
+    parser.add_argument('-e', '--experiment', default=DEFAULT_EXPERIMENT_ID, 
+                        help='Select a valid experiment to run the task for this user')
     args = parser.parse_args()
 
     # Start BCI Main
-    bci_main(args.parameters, str(args.user), int(args.type), str(args.mode))
+    bci_main(args.parameters, str(args.user), str(args.task), str(args.experiment))
```

Comment:
Now that we're passing in the task name rather than the index it seems like it would be easy to pass in a wrong value. We could attempt to get the ExperimentType here so we fail fast if it's missing. It would also simplify the type annotations in the other methods.
  `bci_main(args.parameters, str(args.user), ExperimentType.by_value(args.task), str(args.experiment))`

Reply:
I like the idea of failing faster. I'll do that. In the next PR, I validate the experiment first thing as well in order to fail before any other processes are started.